install.sh

```diff
@@ -447,7 +450,7 @@ fi
 cd "/usr" || exit 1
 
 ####################################################################### script
-USABLE_GIT="$(command -v git)"
+USABLE_GIT="$(find_tool git)"
```

Comment:
This doesn't work on macOS

Reply:
Yes I am on it its a bit tricky tho. The troll operating system macOS puts a git in PATH that is not installed :D
So when running ``git --version`` it prints this "version" ``xcode-select: note: no developer tools were found at '/Applications/Xcode.app', requesting install. Choose an option in the dialog to download the command line developer tools.``

Which does not pass the ``test_git`` version check. The CI was working because later in the install script it does run the xcode install. So I have to think about how to handle this.

Reply:
I haven't even red all lines of the install script. How messy would it be to move the [xcode install lines](https://github.com/Homebrew/install/blob/16aefe107e14322a2104c65bff42de5f6ac6297c/install.sh#L840-L862) before the set of USABLE_GIT?